soilmask.py

```diff
@@ -420,3 +422,4 @@ def perform_process(self, environment: Environment, check_md: dict, transformer_
 if __name__ == "__main__":
     CONFIGURATION = ConfigurationSoilmask()
     entrypoint.entrypoint(CONFIGURATION, SoilMask())
+    sys.exit(0)
```

Comment:
This is redundant and can be removed. A successful Python script will exit with code 0.

Reply:
Changed